specs-go/v1/descriptor.go

```diff
@@ -14,10 +14,12 @@
 
 package v1
 
+import specs "github.com/opencontainers/image-spec/specs-go"
+
 // Descriptor describes the disposition of targeted content.
+// This structure provides `application/vnd.oci.descriptor.v1+json` mediatype when marshalled to JSON
 type Descriptor struct {
-	// MediaType contains the MIME type of the referenced object.
-	MediaType string `json:"mediaType"`
+	specs.MediaTyped
```

Comment:
oh, so this is actually wrong because the mediatype here is the REFERENCED object not itself.